index.html

```diff
@@ -0,0 +1,225 @@
+            Workshops on various topics such as how to craft your resume.
+          </p>
+        </li>
+        <li class="activity-list networking">
+          <img
+            class="activity-image"
+            src="./images/activities/networking.png"
+            alt="lectures icon"
+          />
+          <h4 class="activity-title">Networking</h4>
+          <p class="activity-brief">
+            Network with other Micronauts from around the world.
+          </p>
+        </li>
+        <li class="activity-list tour">
+          <img
+            class="activity-image"
+            src="./images/activities/tour.png"
+            alt="lectures icon"
+          />
+          <h4 class="activity-title">Tour</h4>
+          <p class="activity-brief">
+            Explore the beautiful city of Accra, Ghana's capital.
+          </p>
+        </li>
+      </ul>
+      <div class="register">
+        <button type="button" class="register-button">Register Now</button>
+      </div>
+      <a href="" id="whole-program">SEE THE WHOLE PROGRAM</a>
+    </section>
+
+    <section class="speakers">
+      <div class="speakers-heading">
+        <h3 class="speakers-title">Featured Speakers</h3>
+        <div id="speakers-underline"></div>
+      </div>
+      <ul class="speakers-list">
+        <!-- Speaker Cards Generated Dynamically with JavaScript -->
+      </ul>
+      <div class="more-speakers-button">
+        <button type="button" class="more-speakers">
+          MORE
+          <img
+            id="more-speakers-arrow"
+            class="up-and-down-arrows"
+            src="./images/down-arrow.svg"
+            alt="down arrow"
+          />
+        </button>
+      </div>
+      <div class="less-speakers-button">
+        <button type="button" class="less-speakers">
+          LESS
+          <img
+            id="less-speakers-arrow"
+            class="up-and-down-arrows"
+            src="./images/up-arrow.svg"
+            alt="up arrow"
+          />
+        </button>
+      </div>
+    </section>
+
+    <section class="sponsors" id="sponsors-link">
+      <div class="sponsors-heading">
+        <h5 class="sponsors-title">Sponsors / Partners</h5>
+        <div id="sponsors-underline"></div>
+      </div>
+      <div class="sponsors-logos">
+        <img src="./images/partners/bvmstudio-logo.png" alt="BVM Studio Logo" />
+        <img src="./images/partners/google-logo.png" alt="Google Logo" />
+        <img src="./images/partners/nita-logo.png" alt="Nita Logo" />
+        <img
+          src="./images/partners/microverse-logo.png"
+          alt="Microverse Logo"
+        />
+        <img src="./images/partners/github-logo.png" alt="GitHub Logo" />
+      </div>
+    </section>
+
+    <footer>
+      <img src="./images/ghmm-logo.png" alt="" class="footer-logo" />
+      <div class="footer-text">
+        <p>Built By Benjamin Semah</p>
+        <p>(2021) All Rights Reserved.</p>
+      </div>
+    </footer>
```

Comment:
- [ ] I would suggest updating this section because according to the original design, in the mobile version the last section is the speakers. Please, kindly address this issue because the sponsors and footer should not be displayed in the mobile version :pushpin: :smiley:
![image](https://user-images.githubusercontent.com/81522435/146962005-87aa103f-7ccd-4551-aa25-1a7501e7eb68.png)

- [ ] Also, I think you could add the "More" button in the mobile version as it is shown in the template :raised_hands: :smiley:
![image](https://user-images.githubusercontent.com/81522435/146961979-2d1eed87-4781-4d12-88f1-a89eec70505c.png)